Makefile

```diff
@@ -306,18 +313,16 @@ sub-tag-images-%:
 ###############################################################################
 # Static checks
 ###############################################################################
-## Perform static checks on the code.
 .PHONY: static-checks
+LINT_ARGS := --deadline 5m --max-issues-per-linter 0 --max-same-issues 0
 static-checks:
-	$(DOCKER_RUN) $(CALICO_BUILD) golangci-lint run --deadline 5m
+	$(DOCKER_RUN) $(CALICO_BUILD) golangci-lint run $(LINT_ARGS)
 
 .PHONY: fix
-## Fix static checks
 fix:
 	goimports -w calicoctl/*
 
 .PHONY: install-git-hooks
-## Install Git hooks
```

Comment:
FYI these `##` commands are so that the `make help` magic picks up these targets.